README.md

```diff
@@ -1,63 +1,64 @@
 
-# Importance of Version Control Systems while coding
+# Importance of Version Control Systems while Coding
```

Comment:
Minor syntactical change: 'Coding' should be capitalized in the title. There are a few other minor syntactical and grammatical fixes throughout - I won't comment on most of the others.